packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.styles.ts

```diff
@@ -92,30 +92,24 @@ export const inputWrapperModeStyles: Record<Theme, string> = {
       }
     }
   `,
+  /** token exceptions: background-color value was designated prior to token system */
   [Theme.Dark]: css`
-    color: ${palette.gray.light3};
+    color: ${color.dark.text.primary.default};
     background-color: ${palette.gray.dark4};
```

Comment:
at this point in standardizing components, we'll intentionally continue to leave the `palette.gray.dark4` even though it's not in the token system until design decides on how to proceed

Reply:
Hmmm. Might be premature to tokenize these then. This could potentially change to specific `input` color tokens